splink/duckdb/duckdb_linker.py

```diff
@@ -118,41 +117,40 @@ def __init__(
         # If user has provided pandas dataframes, need to register
         # them with the database, using user-provided aliases
         # if provided or a created alias if not
-
         input_tables = ensure_is_list(input_table_or_tables)
+        input_tables = [
+            duckdb_load_from_file(t) if isinstance(t, str) else t for t in input_tables
+        ]
 
         input_aliases = self._ensure_aliases_populated_and_is_list(
             input_table_or_tables, input_table_aliases
         )
 
-        # 'homogenised' means all entries are strings representing tables
-        homogenised_tables = []
-        homogenised_aliases = []
-
-        default_aliases = all_letter_combos(len(input_tables))
```

Comment:
If we're going back to short & standard aliases I guess we can get rid of `all_letter_combos`?